smartcar/smartcar.py

```diff
@@ -204,12 +208,13 @@ def exchange_refresh_token(self, refresh_token):
         response = requester.call(method, url, data=data, auth=self.auth).json()
         return set_expiration(response)
 
-    def is_compatible(self, vin, scope):
+    def is_compatible(self, vin, scope, country=None):
```

Comment:
should we default this to `'US'`?

Reply:
Functionally its the same because the backend defaults to 'US'. With that being said, not sure if we should default here too or not

Reply:
> Explicit is better than implicit.

 https://www.python.org/dev/peps/pep-0020/

As developer who uses the Smartcar Python SDK I would prefer to have `country='US'`.